src/components/Projects/ProjectsContent/ProjectsCard/ProjectsCard.js

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+// @flow
+import React, { Component } from 'react'
+import { Row, Col } from 'react-bootstrap'
+import './ProjectsCard.css'
+import Fade from '@material-ui/core/Fade'
+import Grow from '@material-ui/core/Grow'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faAngleDown } from '@fortawesome/free-solid-svg-icons'
+import { SMALL_WIDTH } from '../../../../constants'
+
+type Project = {
+  title: string,
+  description: string,
+  image: string,
+  wiki: string
+};
+
+type Props = {
+  index: number,
+  project: Project,
+  show_scrollers: boolean
+};
+
+type State = {
+  large_view: boolean
+};
+
+/** Component class of Projects' card. */
+class ProjectsCard extends Component<Props, State> {
+  project: Project;
+
+  index: number;
+
+  show_scrollers: boolean;
+
+  /**
+   * Class constructor
+   * @param {list} props: List of projects data.
+   */
+  constructor(props: Props) {
+    super(props)
+    this.cardContent = this.cardContent.bind(this)
+    this.imageContent = this.imageContent.bind(this)
+    this.largeView = this.largeView.bind(this)
+    this.viewSizeLarge = this.viewSizeLarge.bind(this)
+    this.smallView = this.smallView.bind(this)
+
+    this.project = props.project
+    this.index = props.index
+    this.show_scrollers = props.show_scrollers
+    this.state = {
+      large_view: this.viewSizeLarge(),
+    }
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', this.updateView)
+  }
+
+  updateView = () => {
+    this.setState({
+      large_view: this.viewSizeLarge(),
+    })
+  }
+
+  viewSizeLarge = () => {
+    if (window.innerWidth > SMALL_WIDTH) {
+      return true
+    }
+    return false
+  }
+
+  joinUsCallback = (link: string) => {
+    window.open(link)
+  }
+
+  /**
+  * Parses path to projects image
+  * @param {string} imgPath: Path to project's image/photo.
+  * @return {path}
+  */
+  tryRequire = (imgPath: string) => {
+    try {
+      return require(`images/projects/${imgPath}`)
```

Comment:
According to Tommy advice it is better to put this first two lines here:
/* eslint-disable import/no-dynamic-require */
/* eslint-disable global-require */